domain/src/main/java/gov/cms/ab2d/domain/Job.java

```diff
@@ -36,6 +38,22 @@
     private JobStatus status;
     private String statusMessage;
     private Integer progress;
+
+    @Pattern(regexp = "^(((Patient)|(ExplanationOfBenefits)),?)*$",
+            message = "_type should contain a string of comma-delimited FHIR resource types; "
+                    + "currently limited to Patient and ExplanationOfBenefits")
     private String resourceTypes;
 
+    @Override
```

Comment:
We're using Project Lombok to avoid having to declare getters, setters, hashCode, equals, etc explicitly. Let's remove these two methods and then use `@EqualsAndHashCode(onlyExplicitlyIncluded = true)` from Lombok instead. Annotate `id` with `@EqualsAndHashCode.Include`